platform/macos/export/export_plugin.cpp

```diff
@@ -2051,10 +2051,12 @@ bool EditorExportPlatformMacOS::has_valid_export_configuration(const Ref<EditorE
 	String architecture = p_preset->get("binary_format/architecture");
 	if (architecture == "universal" || architecture == "x86_64") {
 		if (!ResourceImporterTextureSettings::should_import_s3tc_bptc()) {
+			err += TTR("Cannot export for universal or x86_64 if S3TC BPTC texture format is disabled. Enable it in project settings.") + "\n";
 			valid = false;
 		}
 	} else if (architecture == "arm64") {
 		if (!ResourceImporterTextureSettings::should_import_etc2_astc()) {
+			err += TTR("Cannot export for arm64 if ETC2 ASTC texture format is disabled. Enable it in project settings.") + "\n";
```

Comment:
I wonder why this doesn't take `universal` into account. It's already handled in the branch above obviously, but shouldn't it requires both formats? @bruvzg

Reply:
Yes, universal export should include both formats, so it should be `} else if (architecture == "universal" || architecture == "arm64") {`.

Reply:
That condition would never be reached though as it already checks `universal` in the previous branch.

Should work if it's just `if` and not `else if` on the other hand.

Reply:
Note for OP: Please make modifications by _amending_ the commit, and *force pushing* the changes, so that it stays as a single commit (see [PR workflow](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#modifying-a-pull-request)).